### PR TITLE
Add additional Foundry test coverage

### LINKED ISSUE #319: Add/complete Foundry test coverage
Add or complete unit test coverage for the following contracts:

GhoDiscountRateStrategy
GhoInterestRateStrategy
GhoManager
UiGhoDataProvider
GhoOracle
GhoAToken
GhoStableDebtToken

### PR DESCRIPTION
**Pull request by cedephrase**
Will close #319 once complete.

### CODE REVIEW

File: src/contracts/test/TestEmptyDiscountRateStrategy.t.sol [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+import {EmptyDiscountRateStrategy} from '../facilitators/aave/mocks/EmptyDiscountRateStrategy.sol';
+
+contract TestEmptyDiscountRateStrategy is TestGhoBase {
+  EmptyDiscountRateStrategy emptyStrategy;
+
+  function setUp() public {
+    emptyStrategy = new EmptyDiscountRateStrategy();
+  }
+
+  function testFuzzConstructor(uint256 debtBalance, uint256 discountTokenBalance) public {
```
Comment (Zer0dot) [inline src/contracts/test/TestEmptyDiscountRateStrategy.t.sol:16]: I think this should be named differently right? Maybe `testFuzzRateAlwaysZero` or something

File: src/contracts/test/TestGhoInterestRateStrategy.t.sol [hunk 2]
```diff
@@ -0,0 +1,18 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+import 'forge-std/Test.sol';
+import './TestGhoBase.t.sol';
+
+contract TestGhoInterestRateStrategy is TestGhoBase {
+  function testFuzzConstructor(
```
Comment (Zer0dot) [inline src/contracts/test/TestGhoInterestRateStrategy.t.sol:8]: Same here with the naming

**Review by Zer0dot — APPROVED** [review]
No further comments here!

**Review by miguelmtzinf — APPROVED** [review]

**Comment by miguelmtzinf** [comment]
Great stuff  @cedephrase @Zer0dot !!

**Pull request merged**